ui/redux/actions/search.js

```diff
@@ -85,31 +115,10 @@ export const doSearch = (rawQuery: string, searchOptions: SearchOptions) => (
   cmd(queryWithOptions)
     .then((data: { body: Array<{ name: string, claimId: string }>, poweredBy: string }) => {
       const { body: result, poweredBy } = data;
-      const uris = [];
-      const actions = [];
-
-      result.forEach((item) => {
-        if (item) {
-          const { name, claimId } = item;
-          const urlObj: LbryUrlObj = {};
-
-          if (name.startsWith('@')) {
-            urlObj.channelName = name;
-            urlObj.channelClaimId = claimId;
-          } else {
-            urlObj.streamName = name;
-            urlObj.streamClaimId = claimId;
-          }
-
-          const url = buildURI(urlObj);
-          if (isURIValid(url)) {
-            uris.push(url);
-          }
-        }
-      });
+      const uris = processLighthouseResults(result);
 
+      const actions = [];
       actions.push(doResolveUris(uris));
```

Comment:
we should try batching here, probably easier on the browser even if it's a bit slower (for FYP + related)